### DISCUSSION TITLE
import produtil.setup

### DISCUSSION THREAD

**Discussion by zhumingying** [dtcenter/METplus#1105] [Configuration] [answered]
Did any one run METPlus in Cheyenne after the machine was updated? 

(NPL) cheyenne1:/glade/work/zhuming/METplus4>run_metplus.py -c ./defaults.conf -c ./PointStat_mpas_upa.conf
Traceback (most recent call last):
  File "/gpfs/fs1/work/zhuming/METplus4/ush/run_metplus.py", line 26, in <module>
    import produtil.setup
ModuleNotFoundError: No module named 'produtil'

**Comment by jprestop** [comment, marked as answer]
Hi @zhumingying.  

I see that you are having trouble running METplus on Cheyenne.   

When I run the following on Cheyenne, I get a successful run:

```
module use /glade/p/ral/jntp/MET/METplus/modulefiles
module load metplus/4.0.0
ncar_pylib
run_metplus.py -c /glade/p/ral/jntp/MET/METplus/METplus-4.0.0/parm/use_cases/met_tool_wrapper/PointStat/PointStat.conf -c /glade/p/ral/jntp/jpresto/METplus/cheyenne.jpresto.conf
```

When I run using your files, but overriding the OUTPUT_BASE in my jpresto.cheyenne.conf file, in the following way, I also get a successful run (at least the start of a successful run - it is still running):

`run_metplus.py -c /glade/work/zhuming/METplus4/defaults.conf -c /glade/work/zhuming/METplus4/PointStat_mpas_upa.conf -c /glade/p/ral/jntp/jpresto/METplus/cheyenne.jpresto.conf`

It looks like this is likely an environment issue.  Please check the way you are loading METplus.  Please feel free to use the available module files that have been set up for METplus and are listed on our [Existing Builds](https://dtcenter.org/community-code/metplus/metplus-4-0-existing-builds) page.
